components/cmake/find_dep_packages.cmake

```diff
@@ -16,10 +16,14 @@
 # errors.
 if (USE_KOKKOS)
 
-  # Kokkos will be built in the sharedlibs if Kokkos_ROOT is
-  # unset.
   if (NOT DEFINED ENV{Kokkos_ROOT})
-    set(ENV{Kokkos_ROOT} ${INSTALL_SHAREDPATH})
+    # We should use the kokkos from Trilinos if we are using Trilinos
+    if (USE_TRILINOS)
+      set (ENV{Kokkos_ROOT} ${Trilinos_ROOT})
```

Comment:
One potential issue with this is that when we switch to the preinstalled kokkos, we'll need a new path. But maybe I can change this then by creating something like `Trilinos_Kokkos_ROOT`.

Reply:
@bartgol , those latter two blocks should be `elseif`s , right?

Reply:
Doesn't matter. The calls to find_package return immediately if the pkg was already found anyways...